ndex2/client.py

```diff
@@ -581,9 +720,28 @@ def get_interconnectquery(self, network_id, search_string,
 
     def search_networks(self, search_string="", account_name=None, start=0, size=100, include_groups=False):
         """
+
         Search for networks based on the search_text, optionally limited to networks owned by the specified
         account_name.
 
+        .. code-block:: python
+
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		#search network by key word 'tutorial', return a list of networks with network summary
+		list_networks=my_ndex.search_networks('INSERT KEY WORD HERE')
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client